### PR TITLE
Prepare for release1.1.13

### PR DESCRIPTION
**Pull request by nlcamp**
## Azure IoT Edge PR checklist:

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines and Best Practices
- [x] I have read the [contribution guidelines](https://github.com/azure/iotedge#contributing).
- [x] Title of the pull request is clear and informative.
- [x] Description of the pull request includes a concise summary of the enhancement or bug fix.

### Testing Guidelines
- [ ] Pull request includes test coverage for the included changes.
- Description of the pull request includes 
	- [ ] concise summary of tests added/modified
	- [ ] local testing done.  

### Draft PRs
- Open the PR in `Draft` mode if it is:
	- Work in progress or not intended to be merged.
	- Encountering multiple pipeline failures and working on fixes.

_Note: We use the kodiakhq bot to merge PRs once the necessary checks and approvals are in place. When it merges a PR, kodiakhq converts the PR title to the commit title, PR description to the commit description, and squashes all the commits in the PR to a single commit. The net effect is that entire PR becomes a single commit. Please follow the best practices mentioned [here](https://chris.beams.io/posts/git-commit/#:~:text=The%20seven%20rules%20of%20a%20great%20Git%20commit,what%20and%20why%20vs.%20how%20For%20example%3A%20) for the PR title and description_

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,29 @@
...
+## iotedged
+### Bug Fixes
+* Update Rust `regex` version to address vulnerability [5573eff](https://github.com/Azure/iotedge/commit/5573eff191310e5a226d924b40d1cea035b0a3fd)
+* Publish arm32v7 packages for Debian 9, 10, and 11 to packages.microsoft.com [3e9604b](https://github.com/Azure/iotedge/commit/3e9604b67b6320dba4aab22c4d81b4db67e0c974)
+
+## Diagnostics Module
+### Bug Fixes
+* Update Base Images to address gzip vulnerability [CVE-2022-1271](https://ubuntu.com/security/CVE-2022-1271)
+
+## Metrics Collector
```
Comment (damonbarry) [inline CHANGELOG.md:19]: I think you can leave metrics collector out of this changelog, since it's a separate release with its own changelog.

Comment (nlcamp) [reply CHANGELOG.md]: Ok, I removed that section.

File: edge-modules/metrics-collector/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,12 @@
+# 1.0.7
+### Bug Fixes
+* Update Base Images to address gzip vulnerability [CVE-2022-1271](https://ubuntu.com/security/CVE-2022-1271)
+
+# 1.0.6 (2022-03-15)
+
+## Bug Fixes
+- Update Base Images for a Security Patch [.NET Core 3.1.23 - March 8, 2022](https://github.com/dotnet/core/blob/main/release-notes/3.1/3.1.23/3.1.23.md)
```
Comment (yophilav) [inline edge-modules/metrics-collector/CHANGELOG.md:8]: Nit: `-` should be `*`

File: CHANGELOG.md (hunk 1 above)
Comment (yophilav) [reply CHANGELOG.md]: With that said though, we need to remember to do the release of that independently.

File: edge-modules/metrics-collector/CHANGELOG.md (hunk 2 above)
Comment (nlcamp) [reply edge-modules/metrics-collector/CHANGELOG.md:9]: Looks like the rest of the file uses '-' so I went ahead and left this but changed the 1.0.7 bullet to be consistent.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,25 @@
...
+* Publish arm32v7 packages for Debian 9, 10, and 11 to packages.microsoft.com [3e9604b](https://github.com/Azure/iotedge/commit/3e9604b67b6320dba4aab22c4d81b4db67e0c974)
+
+## Diagnostics Module
+### Bug Fixes
+* Update Base Images to address gzip vulnerability [CVE-2022-1271](https://ubuntu.com/security/CVE-2022-1271)
+
+## Simulated Temperature Sensor
+### Bug Fixes
+* Update Base Images to address gzip vulnerability [CVE-2022-1271](https://ubuntu.com/security/CVE-2022-1271)
+
```
[the remark this answers is not in the document]
Comment (nlcamp) [reply CHANGELOG.md:26]: Fixed.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,25 @@
...
+## Edge Hub
+### Bug Fixes
+* Update Base Images to address gzip vulnerability [CVE-2022-1271](https://ubuntu.com/security/CVE-2022-1271)
+
+## iotedged
+### Bug Fixes
+* Update Rust `regex` version to address vulnerability [5573eff](https://github.com/Azure/iotedge/commit/5573eff191310e5a226d924b40d1cea035b0a3fd)
+* Publish arm32v7 packages for Debian 9, 10, and 11 to packages.microsoft.com [3e9604b](https://github.com/Azure/iotedge/commit/3e9604b67b6320dba4aab22c4d81b4db67e0c974)
```
[the remark this answers is not in the document]
Comment (nlcamp) [reply CHANGELOG.md]: Done.

**Review by yophilav — APPROVED** [review]

**Review by damonbarry — APPROVED** [review]

**Pull request merged**